### PR TITLE
BRMP-12794: GTM code commit, (dup) header and index

### PR DESCRIPTION
**Pull request by pramodganore**

### CODE REVIEW

**Review by dsomalaraju — APPROVED** [review]

**Review by chinchalinchin — APPROVED** [review]

File: frontend/src/index.html [hunk 1]
```diff
@@ -11,8 +11,32 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
+
```
Comment (pramodganore) [inline frontend/src/index.html:14]: duplicate (in addition to the angular route (npm package)

File: frontend/src/app/shared/header/header.component.html [hunk 2]
```diff
@@ -1,3 +1,22 @@
+<script async src="https://www.googletagmanager.com/gtag/js?id={{GTM_TRACKING_ID}}"></script>
+
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', '{{GTM_TRACKING_ID}}', {'anonymizeIp': true}, {'forceSSL': true}, {'tansport': beacon});
```
Comment (pramodganore) [inline frontend/src/app/shared/header/header.component.html:8]: Duplicate in addition to angular npm package. this code seems to vanish when i compile not sure why.

**Review by dsomalaraju — APPROVED** [review]

**Comment by chinchalinchin** [comment]
might want to open a pr for development too.

File: frontend/src/app/shared/header/header.component.html (hunk 2 above)
Comment (ccslowmo) [reply frontend/src/app/shared/header/header.component.html:8]: @pramodganore - embedding <script> tags in angular component html is uncommon, and angular will remove them when it compiles.

**Review by ccslowmo — APPROVED** [review]

**Comment by pramodganore** [comment]
> might want to open a pr for development too.

Will do

**Review by dsomalaraju — APPROVED** [review]

**Review by ccslowmo — CHANGES_REQUESTED** [review]
tag manager is breaking application, will try to fix

**Pull request closed without merging**